addons/beehave/debug/debugger.gd

```diff
@@ -0,0 +1,58 @@
+@tool
+class_name BeehaveEditorDebugger
+extends EditorDebuggerPlugin
+
+const DebuggerTab := preload("debugger_tab.gd")
+var debbuger_tab := DebuggerTab.new()
+
+
+func _has_capture(prefix: String) -> bool:
+	return prefix == "beehave"
+
+
+func _capture(message: String, data: Array, session_id: int) -> bool:
+	if message == "beehave:register_tree":
+		debbuger_tab.register_tree(data[0])
+		return true
+	if message == "beehave:unregister_tree":
+		debbuger_tab.unregister_tree(data[0])
+		return true
+	if message == "beehave:process_tick":
+		debbuger_tab.graph.process_tick(data[0], data[1])
+		return true
+	if message == "beehave:process_begin":
+		debbuger_tab.graph.process_begin(data[0])
+		return true
+	if message == "beehave:process_end":
+		debbuger_tab.graph.process_end(data[0])
+		return true
+	return false
+
+
+func _setup_session(session_id: int) -> void:
+	var session := get_session(session_id)
+	session.started.connect(debbuger_tab.start)
+	session.stopped.connect(debbuger_tab.stop)
+
+	debbuger_tab.name = "🐝Beehave"
```

Comment:
@DarkMessiah does Godot already support emojis in the editor? I remember previously that it had difficulties rendering it.

Reply:
Don't see any problem right now. I can remove emoji :)

Reply:
Leave it in for now, I like emojis!